src/ucp/rndv/rndv_rtr.c

```diff
@@ -235,6 +240,14 @@ static void ucp_proto_rndv_rtr_abort(ucp_request_t *req, ucs_status_t status)
     rpriv->data_received(req, 0);
 }
 
+static void
+ucp_proto_rndv_rtr_reset(ucp_request_t *req)
+{
+    ucp_proto_rndv_rtr_clean(req, UCP_DT_MASK_ALL);
+    ucp_send_request_id_release(req);
+    req->flags &= ~UCP_REQUEST_FLAG_PROTO_INITIALIZED;
```

Comment:
we assume UCP_REQUEST_FLAG_PROTO_INITIALIZED was set?

Reply:
need to double check resource allocation